src/pfe/portal/server.js

```diff
@@ -45,7 +45,9 @@ async function main() {
     let keys = await createCertificateAsync({ selfSigned: true });
     const https = require('https');
     server = https.createServer({ key: keys.serviceKey, cert: keys.certificate }, app).listen(serverPort);
-  } else {
+    console.log("Generated PFE server certificate:")
+    console.log(keys.certificate)
```

Comment:
We probably shouldn't leave `console.log` commands in server.js. (Also is it safe to dump the contents of the certificate to the console or log?)
There are also console.log calls in src/gatekeeper/server.js but I suspect that's because the gatekeeper is too small to bother with a logger. (However the same question applied to dumping the certificate.)